mmv1/products/compute/ForwardingRule.yaml

```
@@ -653,3 +665,15 @@ properties:
     enum_values:
       - 'IPV4'
       - 'IPV6'
+  - name: ipCollection
+    type: String
+    description: |
```

Since this can take a self link or a relative path, but the API always returns a self link, you will need a diff suppress func so that you don't end up with "diff after apply" errors in the tests. (For more information about this kind of diff, see https://googlecloudplatform.github.io/magic-modules/develop/diffs/#normalized-value.) The built-in function for self links is:
 
```suggestion
    diff_suppress_func: 'tpgresource.CompareSelfLinkRelativePaths'
    description: |
```

But that only checks the _full_ relative path; it doesn't support the version that excludes the project. There are two possible solutions to this:

1. Add validation to only allow either full self links or full partial urls
2. Write a custom diff suppress function